Sources/SATSCore/Assets/Colors.xcassets/Signal/signalDeleteText.colorset/Contents.json

```diff
@@ -0,0 +1,38 @@
+{
+  "colors" : [
+    {
+      "color" : {
+        "color-space" : "srgb",
+        "components" : {
+          "alpha" : "1.000",
+          "blue" : "0x26",
+          "green" : "0x32",
+          "red" : "0xD9"
```

Comment:
Reading RGBA colors in alphabetical order (ABGR?) makes my head hurt a little. :grin: